test/starknet/vanilla_space.ts

```diff
@@ -48,31 +49,59 @@ async function setup() {
   };
 }
 
+function setup_calldata(
```

Comment:
camelCase here/

Reply:
[use camelcase; rename setup_calldata to setupProposeCalldata](https://github.com/snapshot-labs/sx-core/pull/54/commits/9a1ad38aecf21c32382abf5a53c2387beff40eb2)

Reply:
Yep, we should use camelCase on all the functions name but also all variables